source/GuiElements.h

```diff
@@ -106,9 +106,18 @@ class WaypointDisplay : public GuiWindow {
 
 class PlayerControls : public GuiWindow {
 protected:
+	
+	int	m_connectedClientIdx = 0;
+
+	const Array<String> m_connectedClients = { "Client 1", "Client 2" };
```

Comment:
I'd like to see this replaced with something more flexible to multiple clients (a dynamically managed array). Also I'd prefer to refer to clients with something more descriptive than just "Client 1" or "Client 2". I know I've talked with @Ben-Peters and @alhayden about referring to clients using either (1) their entered username or (2) their IP address, this would be good to do here.

Reply:
I could not figure out how to update the gui dynamically. (Pointers maybe?) But I'll keep this suggestion for next iteration of this feature. For our current research goals, I believe hard-coded values such as these would work okay.